pkg/model/event.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// EventDefinitionID builds a unique identifier based on the given name and labels.
+// It returns the exact same string as long as both are the same.
+func EventDefinitionID(name string, labels map[string]string) string {
```

Comment:
Actually it will be used by https://github.com/pipe-cd/pipe/issues/1406. It could be increased the need to build a unique identity made of Name and Labels, that's why I decided to make a helper function.

Reply:
I see. Now we are having our cache/store for the event. So maybe we don't need to merge them for simplicity.

The user can merge by changing their configuration if needed.
We implicitly merge them and combine their changes into a single commit that may cause the behavior they were not expected.
(I am not sure that I understood the merging correctly.)

Reply:
>(I am not sure that I understood the merging correctly.)

Your understanding is correct. I attempted to merge it to combine their changes into a single commit.

Reply:
Okay, I agree with all of them. So let's this function private and think it if needed in the future!